<launch>
    <rosparam command="load" file="$(find jetbot_rl_pkg)/config/jetbot_params_deepQ.yaml" />
    Launch the training system
    <node pkg="jetbot_rl_pkg" name="jetbot_gym" type="jetbot_start_multi_train.py" output="screen"/>
</launch>